### ISSUE TITLE
Map required

### ISSUE THREAD

**Issue by Subtixx** [LifeStyleRolePlay/qb-beanmachine#1]
This requires a custom map which is not included, nor mentioned anywhere.

**Comment by LifeStyleRolePlay** [comment]
Yes I bought the map from UncleJust so I cannot share it myself. There is a free MLO on GTA 5 Mods if you'd like to download that one and change the locations

**Comment by Subtixx** [comment]
Can you please mark this in the README.md? And also link there where to buy it?
